en_US/data-integration/data-bridge-kafka.md

```diff
@@ -152,6 +167,92 @@ You can also click **Integration** -> **Flow Designer** to view the topology. Th
 
 ![Kafka_producer_bridge](./assets/Kafka_producer_bridge.png)
 
+### Configure Kafka Dynamic Topics
+
+Starting from EMQX v5.7.2, you can dynamically configure the Kafka topics in the Kafka Producer Sink configuration using the environment variables or variable templates. This section introduces these two use cases in dynamic topic configuration.
+
+#### Use Environment Variables
+
+EMQX v5.7.2 introduces a new functionality of dynamically assigning the values retrieved from [environment variables](../configuration/configuration.md/#environment-variables) to a field within messages during the SQL processing phase. This functionality uses the [getenv](../data-integration/rule-sql-builtin-functions.md#system-function) function from the built-in SQL functions of the rule engine to retrieve environment variables from EMQX. The values of the variables are then set into SQL processing results. As an application of this feature, when configuring Kafka topics in Kafka Sink rule actions, you can reference fields from rule output results to set the Kafka topic. The following is a demonstration of this application:
+
+::: tip Note
+
+To prevent leakage of other system environment variables, the names of environment variables must have a fixed prefix `EMQXVAR_`. For example, if the function variable name read by `getenv` is `KAFKA_TOPIC`, the environment variable name set must be `EMQXVAR_KAFKA_TOPIC`.
```

Comment:
```suggestion
To prevent leakage of other system environment variables, the names of environment variables used by rule engine must have a fixed prefix `EMQXVAR_`. For example, if the variable name read by `getenv` function is `KAFKA_TOPIC`, the environment variable name must be set to `EMQXVAR_KAFKA_TOPIC`.
```